test/riak_kv_schema_tests.erl

```diff
@@ -374,6 +405,21 @@ all_backend_multi_test() ->
     cuttlefish_unit:assert_not_configured(M6, "max_memory"),
     ok.
 
+job_class_enabled_test() ->
+    test_job_class_enabled(riak_core_schema()).
+
+test_job_class_enabled({true, RCSchema}) when erlang:is_list(RCSchema) ->
+    Config = cuttlefish_unit:generate_templated_config(
+        [RCSchema, "../priv/riak_kv.schema"], [],
+        riak_core_schema_tests:context() ++ context()),
+
+    cuttlefish_unit:assert_config(
+        Config, "riak_core.job_accept_class",
+        lists:sort(?DEFAULT_ENABLED_JOB_CLASSES)),
+    ok;
+test_job_class_enabled(_) ->
```

Comment:
I'd like to see a log message here, so that it's clear why the test was skipped.

Even in the "normal" case where it just can't find the riak_core schema, it'd be good to make it explicit why the test is being skipped so that the user has a chance to notice and set things up to correctly run this test.

On the flip side, if some unexpected error occurs, the existing code would also silently swallow it, so it might also be good to have two different log messages: one for the schema-not-found case, and a separate warning for any unexpected errors.